### PR TITLE
Change argument for SDL binary file

### PR DESCRIPTION
**Pull request by OHerasym**
### PR Changes:
- Change argument for SDL binary file

Related: [APPLINK-15026](https://adc.luxoft.com/jira/browse/APPLINK-15026)

### CODE REVIEW

**Comment by OHerasym** [comment]
Please review: @anosach-luxoft , @LuxoftAKutsan , @orhan-mehmedov, @dtrunov

**Comment by dtrunov** [comment]
@OHerasym Reviewed

**Comment by orhan-mehmedov** [comment]
@OHerasym Reviewed

File: README.md [hunk 1]
```diff
@@ -67,11 +67,11 @@ $ sudo find / -name qmake
 
 ### Useful options:
 #### Path to SDL
-You can setup path to SDL via command line with ```--sdl_core``` option.
+You can setup path to SDL via command line with ```--sdl-core``` option.
 
 **Example :**
 ```
-./start.sh --sdl_core=~/development/sdl/build/bin ./test_scripts/ActivationDuringActiveState.lua
+./start.sh --sdl-core=~/development/sdl/build/bin ./test_scripts/ActivationDuringActiveState.lua
```
Comment (LuxoftAKutsan) [inline README.md]: @OHerasym Please take a look on atf_scripts repository to fix this parameter there.

Comment (OHerasym) [inline README.md]: @LuxoftAKutsan , Done

**Comment by LuxoftAKutsan** [comment]
Reviewed

**Pull request merged**